granulate_utils/containers/docker.py

```diff
@@ -35,15 +37,27 @@ def get_runtimes(self) -> List[str]:
         return ["docker"]
 
     @staticmethod
-    def _create_container(container: docker.models.containers.Container) -> Container:
+    def _parse_docker_ts(ts: str) -> Optional[datetime]:
+        assert ts.endswith("Z")  # assert UTC
+        if ts.startswith("0001"):
+            return None
+        return isoparse(ts)
+
+    @classmethod
+    def _create_container(cls, container: docker.models.containers.Container) -> Container:
         pid: Optional[int] = container.attrs["State"].get("Pid")
+        created = cls._parse_docker_ts(container.attrs["Created"])
+        assert created is not None
+        started_at = cls._parse_docker_ts(container.attrs["State"]["StartedAt"])
         if pid == 0:  # Docker returns 0 for dead containers
             pid = None
```

Comment:
Move this code block next to `pid` handling, so there are 3 lines handling `pid` then the lines handling the times stuff and then the `Container` object